docs/tutorial/node-app.rst

```diff
@@ -116,9 +116,22 @@ service:
     2023-10-30T12:37:33.663Z [pebble] Service "app" starting: node server.js
     2023-10-30T12:37:33.864Z [app] Running on http://0.0.0.0:8080
 
-Next, open your web browser and navigate to ``http://localhost:8000``. You
-should see a blank page with a "Hello World from inside the rock!" message.
-Success!
+Next we'll verify that the Node.js app is up and running. If you're running this
+tutorial on a regular Ubuntu system, open your web browser and navigate to
```

Comment:
```suggestion
Next we'll verify that the Node.js app is up and running. If you're working on
a regular Ubuntu system, open your web browser and go to
```

A bit of a nitpick, but "run" isn't the best verb to apply to tutorials.

I find "navigate" to be cumbersome and prefer "go", but I'll leave it up to you to choose.